extensions/cornerstone/src/init.js

```diff
@@ -362,28 +415,31 @@ const _initMeasurementService = (MeasurementService, DisplaySetService) => {
     ArrowAnnotate.toMeasurement
   );
 
-  return csToolsVer4MeasurementSource;
+  return {
+    source: csToolsVer4MeasurementSource,
+    mappings
+  };
 };
 
 const _connectToolsToMeasurementService = (
   MeasurementService,
   DisplaySetService
 ) => {
-  const csToolsVer4MeasurementSource = _initMeasurementService(
+  const { source: csToolsVer4MeasurementSource, mappings } = _initMeasurementService(
     MeasurementService,
     DisplaySetService
   );
   _connectMeasurementServiceToTools(
     MeasurementService,
-    csToolsVer4MeasurementSource
+    csToolsVer4MeasurementSource,
+    mappings
```

Comment:
Doesn't appear to be using this argument in the function?

Reply:
Updated, thanks.